src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -76,4 +86,42 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath) thro
         addressBookStorage.saveAddressBook(addressBook, filePath);
     }
 
+    // ================ Import/ Export Csv methods ==============================
+
+    @Override
+    public void exportProperties(ReadOnlyAddressBook addressBook, File file) throws IOException {
```

Comment:
These methods can go in a separate class to improve code quality (srp/slap/facade pattern)